storybook/src/components/button/button.css-template.ts

```diff
@@ -8,24 +8,30 @@ export const cssButton: ComponentImplementation<Button | ButtonAnchor> = {
   component: "button",
   implementation: "html-css",
   template: ({ iconTemplate, tooltipTemplate }) => {
-    function getClassName(variant: "primary" | "secondary" | "tertiary" | null, modifier: string | undefined) {
-      if (variant && modifier) {
-        return `dso-${variant} ${modifier}`;
-      }
+    function getClassName(
+      variant: "primary" | "secondary" | "tertiary" | null,
+      modifier: string | undefined,
+      mode?: string | undefined
```

Comment:
De `?` in een parametername geeft aan of een parameter optioneel is:

```
function greet(name?: string)
function wave(name: string | undefined)
```

In beide gevallen is `name: string | undefined`. Het verschil is dat je `greet()` mag doen maar `wave(undefined)` moet doen.

Ik denk dat we dat laatste willen: Om de juiste className te bepalen zijn `variant`, `modifier` en `mode` nodig. Dat de className anders is omdat een van deze parameters `undefined` (of `null`) is, is juist een implementatie detail.

Reply:
In dit geval is het zo de we de `mode` niet nodig hebben in de `Button`, maar alleen in de `Anchor`. Daarom optioneel gemaakt. Want ik wil doen wat je voorstelt, moet ik hem ook gebruiken in de `Button`, maar zou dan niet weten hoe. Ideeën? @tfrijsewijk

Reply:
(Gisteren hebben we dit besproken en samen een oplossing gemaakt)